topic2/Main.java

```diff
@@ -0,0 +1,16 @@
+package topic2;
+
+
+public class Main {
+    public static void main(String[] args) {
+        Person marinel = new Person();
+        marinel.fName = "John";
```

Comment:
This kinf of assigments violate inheritance principle, please make the access to class parameters impossible :)

Reply:
Just wanted to show how access modifiers work on Person class. Ok thank you!